api/serverpb/server.proto

```diff
@@ -141,6 +141,16 @@ message SlackAlertingSettings {
   ];
 }
 
+// STTCheckIntervals represents intervals between STT checks.
+message STTCheckIntervals {
+  // Standard check interval.
```

Comment:
should we mention the default duration of the intervals in the comments?

Reply:
I don't think so, defaults defined in other repo. If we will change defaults in the future it will be nice to have all that stuff in one place.